proofs/refiner.mli

```diff
@@ -89,15 +103,28 @@ exception FailError of int * Pp.t Lazy.t
 val catch_failerror  : Exninfo.iexn -> unit
 
 val tclORELSE0       : tactic -> tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclORELSE0"]
 val tclORELSE        : tactic -> tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclORELSE"]
 val tclREPEAT        : tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclREPEAT"]
 val tclFIRST         : tactic list -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclFIRST"]
 val tclTRY           : tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclTRY"]
 val tclTHENTRY       : tactic -> tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclTHENTRY"]
 val tclCOMPLETE      : tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclCOMPLETE"]
 val tclAT_LEAST_ONCE : tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclAT_LEAST_ONCE"]
 val tclFAIL          : int -> Pp.t -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclFAIL"]
 val tclFAIL_lazy     : int -> Pp.t Lazy.t -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclFAIL_lazy"]
 val tclDO            : int -> tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclDO"]
 val tclPROGRESS      : tactic -> tactic
+[@@ocaml.deprecated "Use Tactical.New.tclPROGRESS"]
 val tclSHOWHYPS      : tactic -> tactic
+[@@ocaml.deprecated]
```

Comment:
Missing a string here.

Reply:
There is no alternative for that, it's a tactic that is only used internally by Proof General.

Reply:
Maybe say that?

Reply:
Done.